lib/Statistics/Track/Unsubscribes.php

```diff
@@ -23,7 +23,9 @@ public function __construct(
     $this->statisticsUnsubscribesRepository = $statisticsUnsubscribesRepository;
   }
 
-  public function track(int $subscriberId, int $queueId = null, string $source) {
+  public function track(int $subscriberId, int $queueId = null, string $source, string $meta = null) {
```

Comment:
An optional argument (`queueId`) is followed by a mandatory argument (`source`). 

Ideally, this should not be allowed by our coding standard as we seem to move closer to [PSR-2](https://www.php-fig.org/psr/psr-2/#44-method-arguments):
> Method arguments with default values MUST go at the end of the argument list.